data/h3_data_importer/delete_h3_tables.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import click
+import psycopg
+
+from utils import get_connection_info
+
+log = logging.getLogger("delete_h3_tables")
+logging.basicConfig(level=logging.INFO)
+
+
+@click.command()
+@click.option("--drop-contextuals", is_flag=True)
+@click.option("--dry-run", is_flag=True)
+def main(drop_contextuals: bool, dry_run: bool):
+    with psycopg.connect(get_connection_info()) as conn:
+        with conn.cursor() as cursor:
```

Comment:
n00b question: why do we need a different context here? Does the cursor needs to be closed? Could we explicitly do something like ```cursor.close()``` instead of instantiating a new context?